apps/storybook/src/TiledHeatmap.stories.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  getLevelsSize,
+  PanMesh,
+  VisCanvas,
+  ZoomMesh,
+  TiledHeatmap,
+  TilesApi,
+} from '@h5web/lib';
+import type { ColorMap, Domain, Size } from '@h5web/lib';
+import { ScaleType } from '@h5web/shared';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { clamp } from 'lodash';
+import ndarray from 'ndarray';
+import { useMemo } from 'react';
+import { createFetchStore } from 'react-suspense-fetch';
+import type { Vector2 } from 'three';
+
+import FillHeight from './decorators/FillHeight';
+
+// See https://en.wikipedia.org/wiki/Mandelbrot_set
+function mandelbrot(
+  iterations: number,
+  xDomain: Domain,
+  yDomain: Domain,
+  size: Size
+) {
+  const { width, height } = size;
+
+  const array = ndarray(new Float32Array(height * width), [height, width]);
+  array.data.fill(0);
```

Comment:
To avoid looping through the array twice, could `0` be set as part of the `for` loops below? Maybe before the third nested `for` loop?

You could also put the two domain ranges in constants to save some subtractions.